### PR TITLE
RN-890 Add Data Tables to LESMIS Admin Panel

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:

- Some refactors that didn't end up being necessary but I left in anyway
- Addin two pages to the LESMIS admin panel

### CODE REVIEW

File: packages/lesmis/src/components/MainMenu.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ const AdminPanelLink = () => {
   const { translate } = useI18n();
 
   return (
-    <LocaleListItemLink to="/admin" target="_blank">
+    <LocaleListItemLink to="/admin">
```
Comment (IgorNadj) [inline packages/lesmis/src/components/MainMenu.js:100]: I tend to remove these when I find them, the general UX advice on this: https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187

File: packages/lesmis/src/translations/en.json [hunk 2]
```diff
@@ -264,6 +264,9 @@
     "permissions": "Permissions",
     "users": "Users",
     "user": "User",
-    "onceDeletedThisCantBeUndone": "Once deleted this can't be undone"
+    "onceDeletedThisCantBeUndone": "Once deleted this can't be undone",
```
Comment (IgorNadj) [inline packages/lesmis/src/translations/en.json:267]: Leaving these untranslated, see discussion here: https://beyondessential.slack.com/archives/C02C43K1WN6/p1682984229909659?thread_ts=1682922186.739229&cid=C02C43K1WN6

File: packages/lesmis/src/translations/lo.json [hunk 3]
```diff
@@ -260,6 +260,9 @@
     "permissions": "ການອະນຸຍາດ",
     "user": "ຜູ້ໃຊ້",
     "users": "ຜູ້ໃຊ້",
-    "onceDeletedThisCantBeUndone": "ເມື່ອລຶບແລ້ວອັນນີ້ຈະບໍ່ສາມາດຍົກເລີກໄດ້"
+    "onceDeletedThisCantBeUndone": "ເມື່ອລຶບແລ້ວອັນນີ້ຈະບໍ່ສາມາດຍົກເລີກໄດ້",
+    "externalData": "External Data",
+    "externalDatabaseConnections": "External Database Connections",
+    "dataTables": "Data Tables"
```
Comment (IgorNadj) [inline packages/lesmis/src/translations/lo.json:266]: Leaving these untranslated, see discussion here: https://beyondessential.slack.com/archives/C02C43K1WN6/p1682984229909659?thread_ts=1682922186.739229&cid=C02C43K1WN6

File: packages/central-server/src/index.js [hunk 4]
```diff
@@ -63,7 +63,7 @@ import winston from './log';
   /**
    * Start the server
    */
-  const port = 8090;
+  const port = process.env.PORT || 8090;
```
Comment (rohan-bes) [inline packages/central-server/src/index.js:66]: Oh! Nice catch 👍

**Review by rohan-bes — APPROVED** [review]
Nice work, thanks @IgorNadj

**Pull request merged**